proto/cosmos/orm/v1/schema.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package cosmos.orm.v1alpha1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/orm/types/ormpb";
+
+// SchemaDescriptor describes an ORM schema that contains all the information
+// needed for a dynamic client to decode the stored data.
+message SchemaDescriptor {
+
+  // files is the set of all FileDescriptorProto's needed to decode the stored data.
+  // A schema imposes the constraint that every file and every table within that
+  // schema have at most one instance in the store.
+  google.protobuf.FileDescriptorSet files = 1;
+
+  // modules is the set of modules in the schema.
+  repeated ModuleEntry modules = 2;
+
+  // ModuleEntry describes a single module's schema.
+  message ModuleEntry {
```

Comment:
Currently the new codegen does not support embedded messages like this one and file entry, would it be possible to pull those out?

Reply:
Oh sure. But can we address that on pulsar because I think we'll have problems lots of places then?

Reply:
solved in pulsar